### ISSUE TITLE
need to be ported to mysqli as mysql_ is no longer supported or deprecated

### ISSUE THREAD

**Issue by alfieo** [Danter14/LastTrav-T4#12]
This won't install anymore since the mysql connector is no longer supported and is deprecated so it must be mysqli connector

**Comment by Danter14** [comment]
hello, yes the source code works under php 5.6 it is not updated on the new php

**Comment by alfieo** [comment]
Ok, Thank you so no chance to test it on php 5.6 up then. So it's going to be I believe

**Comment by alfieo** [comment]
Do you have any plans in doing a rework, maybe I can help in finding more bugs too. I can maybe assist in basic bug fixing but not with doing full scripting from scratch. It's a bit mind-blowing though ^_^

**Comment by Danter14** [comment]
if you want to help with the bugs it is with pleasure.  for the redesign of the scripts I must see its to put it under php 8 but it will require a lot of work.

**Comment by alfieo** [comment]
https://github.com/FredZockt/TravianC <--- is ported to mysqli but not completely. Install page creates error. So maybe some of the mysqli connecter commands are not properly set.

**Comment by Danter14** [comment]
yes surely but other problems are mandatory if you try to install under php 8 many variables are no longer supported or removed in this version.

**Comment by darkloveir** [comment]
https://github.com/Shadowss/TravianZ
